include/circt/Dialect/MSFT/PlacementDB.h

```diff
@@ -1,4 +1,5 @@
-//===- PlacementDB.h - Device database -----------------------------*- C++ -*-===//
+//===- PlacementDB.h - Device database -----------------------------*- C++
```

Comment:
This doesn't look right...

Reply:
Good catch. I also somehow missed updating the include guard name. Fixing...